test/MUIDataTable.test.js

```diff
@@ -800,6 +810,14 @@ describe('<MUIDataTable />', function() {
     assert.deepEqual(state.data[0].data[2], 'Las Vegas');
   });
 
+  it('should call onTableInit when MUIDataTable is initialized', () => {
```

Comment:
This is great! But I think it would be nice to also trigger an update and ensure that the call count for `onTableInit` remains `1`. Something from one of the tests to table update could serve well here, such as:

```
    instance.selectRowUpdate('head', 0);
    shallowWrapper.update();
```

Reply:
This is a great suggestion. Added this: 

```
  it('should call onTableInit only 1 time when creating table and calling selectRowUpdate method with type=head', () => {
    const options = { selectableRows: true, onTableInit: spy() };
    const shallowWrapper = shallow(<MUIDataTable columns={columns} data={data} options={options} />).dive();
    const instance = shallowWrapper.instance();

    instance.selectRowUpdate('head', 0);
    shallowWrapper.update();

    assert.strictEqual(options.onTableInit.callCount, 1);
  });
```